Formula/libva.rb

```diff
@@ -1,12 +1,8 @@
 class Libva < Formula
   desc "Hardware accelerated video processing library"
   homepage "https://freedesktop.org/wiki/Software/vaapi/"
-  url "https://www.freedesktop.org/software/vaapi/releases/libva/libva-1.7.3.tar.bz2"
-  sha256 "22bc139498065a7950d966dbdb000cad04905cbd3dc8f3541f80d36c4670b9d9"
-
-  bottle do
-    sha256 "202cad95248823257d1d9cedefb76fc95593ef3dd61d93721a14f000f75bd0db" => :x86_64_linux
-  end
```

Comment:
No need to remove the `bottle` block.

Reply:
Looks like it has to be removed though:
```
==> Installing linuxbrew/xorg/mesa dependency: linuxbrew/xorg/libva
==> Downloading https://linuxbrew.bintray.com/bottles-xorg/libva-1.8.2.x86_64_linux.bottle.tar.gz
/usr/bin/curl --show-error --user-agent Linuxbrew/1.5.6-14-g42f6c7a (Linux; x86_64 Ubuntu 14.04.5 LTS) curl/7.35.0 --fail --location --remote-time --continue-at - --output /home/linuxbrew/.cache/Homebrew/libva-1.8.2.x86_64_linux.bottle.tar.gz.incomplete https://linuxbrew.bintray.com/bottles-xorg/libva-1.8.2.x86_64_linux.bottle.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 401 Unauthorized
Error: Failed to download resource "libva"
Download failed: https://linuxbrew.bintray.com/bottles-xorg/libva-1.8.2.x86_64_linux.bottle.tar.gz
```

Reply:
Hmm. Interesting. I guess because this PR is modifying both `libva` and `mesa`? I wonder how this issue is resolved by upstream. In that case, your workaround of deleting the bottle block is fine.